sections/02_intro.md

```diff
@@ -26,8 +26,73 @@ by "deep" learning when most methods use few hidden layers.*
 
 ### Types of biological problems
 
-This section could contain the types of biological problems people have already applied deep learning to, based on the existing literature.  This could be mentioned in two ways: 
+In this review, we are interested in the application of deep learning to 
+topics of biomedical importance. This covers a large range of topics, which 
+we divide into three broad classes based on their applied areas. We then 
+briefly introduce the types of questions, approaches and data which are 
+typical for each class in the application of deep learning.
 
-1. The type of input data: DNA sequence, protein sequence, clinical data, imaging data, etc.
-2. The output/objective of the model: protein structure/function, protein localization, miRNA/TF target prediction, patient diagnosis/prognosis, drug discovery/prediction, or unsupervised clustering, etc.
+#### Disease and Patient Categorization
 
+One important topic in the biomedical field is the accurate classification of 
+diseases and disease subtypes. In the oncology field, current "gold standard" 
+approaches are limited to either histological approaches, requiring manual 
+human expertise, or shallow molecular markers, such as the cell surface 
+receptors or small panels of genes. One example is the current PAM50 approach 
+in classifying breast cancer, which utilizes the expression of 50 marker 
+genes in order to divide breast cancer patients into six subtypes. 
+Significant heterogeneity still remains within these six subtypes (Mayer et 
+al. 2014)[@doi:10.1158/1078-0432.CCR-13-0583]. Given the increasing wealth of 
+molecular data available, it seems that a more comprehensive subtyping is 
+possible.
+
+Several studies have used deep learning methods in order to better categorize
+breast cancer patients. For example, Tan et al. applied denoising
+autoencoders (DA), an unsupervised approach, in order to cluster breast
+cancer patients (Tan et al. 2014)[@doi:10.1142/9789814644730_0014]. Ciresan
+et al. utilized convolutional neural networks (CNN) to count mitotic
+divisions in histological images; a feature which is highly correlated with
+disease outcome (Ciresan et al. 2013)[@doi:10.1007/978-3-642-40763-5_51].
+Despite these recent advances, a number of challenges exist in this area of
+research, such as the integration of disparate types of data, including
+electronic health records (EHR), imaging and histology data and molecular
+omics data.
+
+#### Fundamental Biological Study
+
+Broadly speaking, topics in this class aim to answer more fundamental 
+biological questions. Deep learning is especially suited in leveraging the 
+large amounts of data from high throughput omics studies. The development of 
+deep learning techniques and complex network architectures allow researchers 
+to answer fundamental biological questions with unprecedented accuracy. One 
```

Comment:
We need to determine the overall message we wish to send.  That will set the tone here for whether we want to claim unprecedented accuracy or something more moderate with great future potential.